docs/pages/how_to_work_with_us.js

```diff
@@ -67,6 +67,37 @@ We ask that any ideas that change the functionality of an existing Gestalt compo
   </MainSection>,
 );
 
+card(
+  <MainSection name="How can you make design contributions to Gestalt?">
+    <MainSection.Subsection
+      description={`
+We ask that any ideas that change the functionality of an existing Gestalt component or introduce a net-new component go through the following process:
+
+1. **Present your idea or suggestion during our Office Hour meetings**
+    - Sign up for an Office Hours slot! Explain project goals and show multiple options explored with and without Gestalt components to help us understand your needs. This does not need to be a polished presentation, but should have enough detail that we understand the ask and why it is needed.
+    - Coordinate with other designers if there are overlapping needs to help us better prioritize in our roadmap.
+    - Be able to explain how this component should be used or not used. This helps us in testing the component against existing components and product surfaces.
+    - Have a rough project timeline available. This allows us to determine if we’re able to accommodate the work.
+2. **Create a Branch in our main file following the steps**
+    OK, great. You went to office hours and we all agree on what we're making. Now it's time to start bringing your designs into Gestalt.
+
+    - Create a Branch file in our main file. New to Branching? Don't worry, our design team will support you with that.
+    - Follow the name convention: Your initials + Component name +What you are doing, for example: CR - Tab: Adding a badge variant.
```

Comment:
_Follow the name convention: Your initials + Component name +What you are doing, for example: CR - Tab: Adding a badge variant._ 

Minor minor thing: Just add a space after the "+" on "What you are doing".

It looks great! Thanks for the addition!

Reply:
Good catch! Done!